src/test/java/com/google/cloud/spanner/pgadapter/ParserTest.java

```diff
@@ -305,7 +305,7 @@ public void testTimestampParsingTimestampValidityChecks() {
   public void testBinaryParsing() {
     ByteArray value = ByteArray.copyFrom(new byte[] {(byte) 0b01010101, (byte) 0b10101010});
     byte[] byteResult = {(byte) 0b01010101, (byte) 0b10101010};
-    byte[] stringResult = {'U', '\\', '2', '5', '2'};
+    byte[] stringResult = {'\\', 'x', '5', '5', 'a', 'a'};
```

Comment:
Changed to hex format from the old escape format for bytea values in text format.